README.md

```diff
@@ -50,7 +50,9 @@ a place to host the server itself:
        "triagers": ["jlipps", "penguinho"],
        "labels": ["AutoAssigned"]
      }
-   ]}
+   ],
+   "bot": "triager-bot"
+   }
```

Comment:
This is so they can @<someone> please close. Not that it matters, but it will prevent confusion (instead of a user just saying 'please close.')